common/common.go

```diff
@@ -180,6 +182,13 @@ func WithNetGRPCOptions(opts ...grpc.ServerOption) NetOption {
 	}
 }
 
+func WithNetPubSub() NetOption {
```

Comment:
One more nit: Can you make this more like the debug option that makes using an upstream `bool` a little nicer, as in: `WithNetPubSub(enabled bool)`

Reply:
Fixed